main.go

```diff
@@ -73,17 +73,17 @@ func shellInDir(dir, cmd string) {
 	shell("cd %s;%s", coalesce(dir, "."), cmd)
 }
 
-func NewWindow(session *Session, dir string) {
+func NewWindow(session *Session, window *Window, dir string) {
 	if session.started {
-		shell("tmux new-window -d -t %s -c %s", session.Name, dir)
+		shell("tmux new-window -d -t %s -n %s -c %s", session.Name, window.Name, dir)
 	} else {
-		shell("tmux new-session -d -s %s -c %s", session.Name, dir)
+		shell("tmux new-session -d -s %s -n %s -c %s", session.Name, window.Name, dir)
 		session.started = true
 	}
```

Comment:
Let's try to gracefully handle the name not being set. How about something like this:
```Go
    var namedWindow string
    if len(window.Name) > 0 {
        namedWindow = fmt.Sprintf(` -n "%s"`, window.Name)
    }

    if session.started {
        shell(`tmux new-window -d -t %s%s -c %s`, session.Name, namedWindow, dir)
    } else {
        shell(`tmux new-session -d -s %s%s -c %s`, session.Name, namedWindow, dir)
        session.started = true
    }
```

Reply:
Clever! I'll try it.